### ISSUE TITLE
Bug: Svg render wrong Position

### ISSUE THREAD

**Issue by edeink** [pixijs/pixijs#10511]
### Current Behavior

<img width="1536" alt="image" src="https://github.com/pixijs/pixijs/assets/10323795/46dfd639-72a7-49a3-8c74-2c3e9c88caa9">

Svg img not render at position (0, 0). (version 8.0). 
Part of tiger is invisible.

### Expected Behavior

<img width="1589" alt="image" src="https://github.com/pixijs/pixijs/assets/10323795/7b9abf00-6537-4187-9a22-78e98f35ab99">

Render in right place, such as version 7.0 did.

### Steps to Reproduce

Open playground url, can get it.

[https://pixijs.com/8.x/playground](https://pixijs.com/8.x/playground?state=eyJmaWxlcyI6eyIvc3JjL2luZGV4LmpzIjp7ImNvZGUiOiJpbXBvcnQgeyBBcHBsaWNhdGlvbiwgQXNzZXRzLCBHcmFwaGljcyB9IGZyb20gJ3BpeGkuanMnO1xuXG4oYXN5bmMgKCkgPT5cbntcbiAgICAvLyBDcmVhdGUgYSBuZXcgYXBwbGljYXRpb25cbiAgICBjb25zdCBhcHAgPSBuZXcgQXBwbGljYXRpb24oKTtcblxuICAgIC8vIEluaXRpYWxpemUgdGhlIGFwcGxpY2F0aW9uXG4gICAgYXdhaXQgYXBwLmluaXQoeyBhbnRpYWxpYXM6IHRydWUsIHJlc2l6ZVRvOiB3aW5kb3cgfSk7XG5cbiAgICAvLyBBcHBlbmQgdGhlIGFwcGxpY2F0aW9uIGNhbnZhcyB0byB0aGUgZG9jdW1lbnQgYm9keVxuICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kQ2hpbGQoYXBwLmNhbnZhcyk7XG5cbiAgICBjb25zdCB0aWdlclN2ZyA9IGF3YWl0IEFzc2V0cy5sb2FkKHtcbiAgICAgICAgc3JjOiAnaHR0cHM6Ly9waXhpanMuY29tL2Fzc2V0cy90aWdlci5zdmcnLFxuICAgICAgICBkYXRhOiB7XG4gICAgICAgICAgICBwYXJzZUFzR3JhcGhpY3NDb250ZXh0OiB0cnVlLFxuICAgICAgICB9LFxuICAgIH0pO1xuXG4gICAgY29uc3QgZ3JhcGhpY3MgPSBuZXcgR3JhcGhpY3ModGlnZXJTdmcpO1xuXG4gICAgYXBwLnN0YWdlLmFkZENoaWxkKGdyYXBoaWNzKTtcbn0pKCk7XG4ifSwiL3NyYy9zdHlsZXMuY3NzIjp7ImNvZGUiOiJib2R5IHsgbWFyZ2luOiAwOyBvdmVyZmxvdzogaGlkZGVuOyB9In0sIi9pbmRleC5odG1sIjp7ImNvZGUiOiJcbjwhRE9DVFlQRSBodG1sPlxuICA8aHRtbD5cbiAgPGhlYWQ%2BXG4gIDx0aXRsZT5QaXhpSlMgUGxheWdyb3VuZDwvdGl0bGU%2BXG4gIDxtZXRhIGNoYXJzZXQ9XCJVVEYtOFwiIC8%2BXG4gIDxsaW5rIHJlbD1cInN0eWxlc2hlZXRcIiBocmVmPVwic3JjL3N0eWxlcy5jc3NcIj5cbjwvaGVhZD5cbjxib2R5PlxuICA8c2NyaXB0IHNyYz1cInNyYy9pbmRleC5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuPC9odG1sPiJ9LCIvcGFja2FnZS5qc29uIjp7ImNvZGUiOiJ7XG4gIFwic2NyaXB0c1wiOiB7XG4gICAgXCJzdGFydFwiOiBcInBhcmNlbCBpbmRleC5odG1sIC0tb3BlblwiLFxuICAgIFwiYnVpbGRcIjogXCJwYXJjZWwgYnVpbGQgaW5kZXguaHRtbFwiXG4gIH0sXG4gIFwiZGVwZW5kZW5jaWVzXCI6IHtcbiAgICBcInBpeGkuanNcIjogXCI4LjEuMFwiXG4gIH0sXG4gIFwiZGV2RGVwZW5kZW5jaWVzXCI6IHtcbiAgICBcInBhcmNlbC1idW5kbGVyXCI6IFwiXjEuNi4xXCIsXG4gICAgXCJAYmFiZWwvY29yZVwiOiBcIl43LjIxLjNcIixcbiAgICBcIkBiYWJlbC9wbHVnaW4tcHJvcG9zYWwtY2xhc3MtcHJvcGVydGllc1wiOiBcIl43LjEwLjFcIlxuICB9LFxuICBcIm1haW5cIjogXCIvaW5kZXguaHRtbFwiXG59In0sIi8uYmFiZWxyYyI6eyJjb2RlIjoie1xuICBcInByZXNldHNcIjogW1xuICAgIFtcbiAgICAgIFwiQGJhYmVsL3ByZXNldC1lbnZcIixcbiAgICAgIHtcbiAgICAgICAgXCJ1c2VCdWlsdEluc1wiOiBcImVudHJ5XCIsXG4gICAgICAgIFwiY29yZWpzXCI6IFwiMy4yOVwiLFxuICAgICAgICBcInRhcmdldHNcIjogXCJsYXN0IDIgY2hyb21lIHZlcnNpb25zXCJcbiAgICAgIH1cbiAgICBdXG4gIF0sXG4gIFwicGx1Z2luc1wiOiBbXG4gICAgXCJAYmFiZWwvcGx1Z2luLXByb3Bvc2FsLWNsYXNzLXByb3BlcnRpZXNcIlxuICBdXG59In0sIi9zYW5kYm94LmNvbmZpZy5qc29uIjp7ImNvZGUiOiJ7XG4gICAgICAgICAgICAgICAgXCJpbmZpbml0ZUxvb3BQcm90ZWN0aW9uXCI6IGZhbHNlXG4gICAgICAgICAgICB9In19LCJ2aXNpYmxlRmlsZXMiOlsiL3NyYy9pbmRleC5qcyJdLCJhY3RpdmVGaWxlIjoiL3NyYy9pbmRleC5qcyJ9&exampleId=custom)

### Environment

- **`pixi.js` version**: *e.g. ^8.1.0*
- **Browser & Version**: *Both Stable Browser (Fireforx, Chrome, Safari, etc)*
- **OS & Version**: *Mac Os 14.4.1*
- **Running Example**: [https://pixijs.com/8.x/playground](https://pixijs.com/8.x/playground?state=eyJmaWxlcyI6eyIvc3JjL2luZGV4LmpzIjp7ImNvZGUiOiJpbXBvcnQgeyBBcHBsaWNhdGlvbiwgQXNzZXRzLCBHcmFwaGljcyB9IGZyb20gJ3BpeGkuanMnO1xuXG4oYXN5bmMgKCkgPT5cbntcbiAgICAvLyBDcmVhdGUgYSBuZXcgYXBwbGljYXRpb25cbiAgICBjb25zdCBhcHAgPSBuZXcgQXBwbGljYXRpb24oKTtcblxuICAgIC8vIEluaXRpYWxpemUgdGhlIGFwcGxpY2F0aW9uXG4gICAgYXdhaXQgYXBwLmluaXQoeyBhbnRpYWxpYXM6IHRydWUsIHJlc2l6ZVRvOiB3aW5kb3cgfSk7XG5cbiAgICAvLyBBcHBlbmQgdGhlIGFwcGxpY2F0aW9uIGNhbnZhcyB0byB0aGUgZG9jdW1lbnQgYm9keVxuICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kQ2hpbGQoYXBwLmNhbnZhcyk7XG5cbiAgICBjb25zdCB0aWdlclN2ZyA9IGF3YWl0IEFzc2V0cy5sb2FkKHtcbiAgICAgICAgc3JjOiAnaHR0cHM6Ly9waXhpanMuY29tL2Fzc2V0cy90aWdlci5zdmcnLFxuICAgICAgICBkYXRhOiB7XG4gICAgICAgICAgICBwYXJzZUFzR3JhcGhpY3NDb250ZXh0OiB0cnVlLFxuICAgICAgICB9LFxuICAgIH0pO1xuXG4gICAgY29uc3QgZ3JhcGhpY3MgPSBuZXcgR3JhcGhpY3ModGlnZXJTdmcpO1xuXG4gICAgYXBwLnN0YWdlLmFkZENoaWxkKGdyYXBoaWNzKTtcbn0pKCk7XG4ifSwiL3NyYy9zdHlsZXMuY3NzIjp7ImNvZGUiOiJib2R5IHsgbWFyZ2luOiAwOyBvdmVyZmxvdzogaGlkZGVuOyB9In0sIi9pbmRleC5odG1sIjp7ImNvZGUiOiJcbjwhRE9DVFlQRSBodG1sPlxuICA8aHRtbD5cbiAgPGhlYWQ%2BXG4gIDx0aXRsZT5QaXhpSlMgUGxheWdyb3VuZDwvdGl0bGU%2BXG4gIDxtZXRhIGNoYXJzZXQ9XCJVVEYtOFwiIC8%2BXG4gIDxsaW5rIHJlbD1cInN0eWxlc2hlZXRcIiBocmVmPVwic3JjL3N0eWxlcy5jc3NcIj5cbjwvaGVhZD5cbjxib2R5PlxuICA8c2NyaXB0IHNyYz1cInNyYy9pbmRleC5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuPC9odG1sPiJ9LCIvcGFja2FnZS5qc29uIjp7ImNvZGUiOiJ7XG4gIFwic2NyaXB0c1wiOiB7XG4gICAgXCJzdGFydFwiOiBcInBhcmNlbCBpbmRleC5odG1sIC0tb3BlblwiLFxuICAgIFwiYnVpbGRcIjogXCJwYXJjZWwgYnVpbGQgaW5kZXguaHRtbFwiXG4gIH0sXG4gIFwiZGVwZW5kZW5jaWVzXCI6IHtcbiAgICBcInBpeGkuanNcIjogXCI4LjEuMFwiXG4gIH0sXG4gIFwiZGV2RGVwZW5kZW5jaWVzXCI6IHtcbiAgICBcInBhcmNlbC1idW5kbGVyXCI6IFwiXjEuNi4xXCIsXG4gICAgXCJAYmFiZWwvY29yZVwiOiBcIl43LjIxLjNcIixcbiAgICBcIkBiYWJlbC9wbHVnaW4tcHJvcG9zYWwtY2xhc3MtcHJvcGVydGllc1wiOiBcIl43LjEwLjFcIlxuICB9LFxuICBcIm1haW5cIjogXCIvaW5kZXguaHRtbFwiXG59In0sIi8uYmFiZWxyYyI6eyJjb2RlIjoie1xuICBcInByZXNldHNcIjogW1xuICAgIFtcbiAgICAgIFwiQGJhYmVsL3ByZXNldC1lbnZcIixcbiAgICAgIHtcbiAgICAgICAgXCJ1c2VCdWlsdEluc1wiOiBcImVudHJ5XCIsXG4gICAgICAgIFwiY29yZWpzXCI6IFwiMy4yOVwiLFxuICAgICAgICBcInRhcmdldHNcIjogXCJsYXN0IDIgY2hyb21lIHZlcnNpb25zXCJcbiAgICAgIH1cbiAgICBdXG4gIF0sXG4gIFwicGx1Z2luc1wiOiBbXG4gICAgXCJAYmFiZWwvcGx1Z2luLXByb3Bvc2FsLWNsYXNzLXByb3BlcnRpZXNcIlxuICBdXG59In0sIi9zYW5kYm94LmNvbmZpZy5qc29uIjp7ImNvZGUiOiJ7XG4gICAgICAgICAgICAgICAgXCJpbmZpbml0ZUxvb3BQcm90ZWN0aW9uXCI6IGZhbHNlXG4gICAgICAgICAgICB9In19LCJ2aXNpYmxlRmlsZXMiOlsiL3NyYy9pbmRleC5qcyJdLCJhY3RpdmVGaWxlIjoiL3NyYy9pbmRleC5qcyJ9&exampleId=custom)

### Possible Solution

`Assets.load(SvgUrl)` not support `viewBox` and `transform`.

<img width="1382" alt="image" src="https://github.com/pixijs/pixijs/assets/10323795/07097017-3d59-4a8d-a91e-b449a4e2f4db">

If I remove those Svg attribute, will get the same result.
<img width="1473" alt="image" src="https://github.com/pixijs/pixijs/assets/10323795/f1611d76-06d6-4868-b494-40ad4d0c3ef4">

**Comment by edeink** [comment]
Found the solution. I should reset graphic position, use:

 `graphic.position.set(-graphic.bounds.minX, -graphic.position.minY)`

<img width="1444" alt="image" src="https://github.com/pixijs/pixijs/assets/10323795/4762e313-eb4c-4c8b-ba44-2fdcbff672ea">

closed.

**Comment by edeink** [comment]
Additionally, the following are my thoughts on debugging:
I observed that Pixi.js correctly detects hits on SVG. Therefore, I debugged the `hittest` function.
Subsequently, I discovered that the graphics context overrides the hittest function.

In other words, all SVG information can be accessed. This makes it easier to determine the expected position.

<img width="1074" alt="image" src="https://github.com/pixijs/pixijs/assets/10323795/901f0fcf-403f-4ccc-9dc0-9baaf6fd0c7e">
